src/Microsoft.CSharp/tests/ArrayHandling.cs

```diff
@@ -31,25 +32,31 @@ public void ArrayTypeNames()
             d = new int[3];
             ex = Assert.Throws<RuntimeBinderException>(() => { string s = d; });
             Assert.Contains("int[]", ex.Message);
+        }
 
-            d = new int[3, 2, 1];
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Multi-dimensional array not supported")]
```

Comment:
Can we include issue ID's
[SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Multi-dimensional array not supported": https://github.com/dotnet/corert/issues/3331")]